ballerina-tests/Ballerina.toml

```
@@ -1,7 +1,7 @@
 [package]
 org = "ballerina"
 name = "websub_tests"
-version = "2.8.1"
+version = "2.9.0"
```

```suggestion
version = "2.9.0"

[platform.java11]
graalvmCompatible = true
```
Lets mark the test package as GraalVM compatible otherwise we will be getting some warnings when running tests

---

ack

---

Fixed with [ffb768a0a1ca4304846d339ec358f3e16486dc5b](https://github.com/ballerina-platform/module-ballerina-websub/pull/604/commits/ffb768a0a1ca4304846d339ec358f3e16486dc5b)